modules/api/pkg/handler/v2/routes_v2.go

```diff
@@ -1439,6 +1439,10 @@ func (r Routing) RegisterV2(mux *mux.Router, oidcKubeConfEndpoint bool, oidcCfg
 		Path("/quotas/{quota_name}").
 		Handler(r.getResourceQuota())
 
+	mux.Methods(http.MethodGet).
+		Path("/totalquota").
```

Comment:
This endpoint's name stands alone a bit. Maybe instead of introducing a new endpoint we could modify existing GET `/quotas` endpoint by introducing additional parameter e.g. `?accumulate=true`. I glanced through the code and it seems possible. With this new approach, using other existing query params for this endpoint, we could get calculation of per project resource quota for free.